sdk/keyvault/azure-security-keyvault-certificates/test/samples/certificate-basic-operations/certificate_basic_operations.cpp

```diff
@@ -67,7 +67,9 @@ int main()
       // start the create process
       auto response = certificateClient.StartCreateCertificate(certificateName, options);
       // wait for complete to get the certificate
-      certificate = response.PollUntilDone(defaultWait).Value;
+      auto pollResponse = response.PollUntilDone(defaultWait).Value;
```

Comment:
We might not need to get the pollResponse now if there's no use for it

```suggestion
      response.PollUntilDone(defaultWait);
```